hotel_housekeeping/wizard/housekeeping_wizard.py

```diff
@@ -5,12 +5,13 @@
 
 class HotelHousekeepingWizard(models.TransientModel):
     _name = "hotel.housekeeping.wizard"
+    _description = "hotel.housekeeping.wizard"
```

Comment:
Same as above, improve _description

Reply:
addressed it ?

Reply:
Yes Sir. already changed it in the last commit.